src/controllers/content.controller.ts

```
@@ -445,6 +445,14 @@ export class contentController {
     @Query() { limit = 5 },
   ) {
     try {
+      // Add the check for the limit
+      let limit = 0
```

ESLint

[error] Insert `;`

(prettier/prettier)